lib/src/modules/apply.cc

```diff
@@ -78,12 +78,14 @@ cli_settings_pluginsync = [
   args['crt'],
   '--hostprivkey',
   args['key'],
-  '--hostcrl',
-  args['crl'],
   '--server_list',
   server_list
 ]
 
+if args['crl']
+  cli_settings_pluginsync << '--hostcrl' << args['crl']
```

Comment:
This will always be required. We should error out in the CPP module code if apply action is requested.

Reply:
I think for this PR we just need to make sure there is no error when we load config, but it *does* error when we try to use apply module action.

Reply:
Yep, that sounds right to me